admin/pr_as_branch

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+set -e
+set -u
+
+
+# Usage
+
+usage() {
+  program=$(basename "$0");
+
+  if [ $# != 0 ]; then echo "$@"; echo ""; fi;
+
+  echo "${program}: usage:";
+  echo "    ${program} <pull_request_number> <trac_ticket_number> <branch_name>";
+
+}
+
+
+# Options
+
+while [ $# != 0 ]; do
+  case "$1" in
+    --help)
+      usage;
+      exit 0;
+      ;;
+    --|*) break; ;;
+  esac;
+done;
+
+if [ $# != 3 ]; then
+    usage "Invalid arguments: $*";
+    exit 1;
+fi;
+
+    PR_NUMBER="$1"; shift;
+TICKET_NUMBER="$1"; shift;
+  BRANCH_NAME="$1"; shift;
+
+
+# Do The Right Thing
+
+#repo="https://github.com/twisted/twisted.git";
+repo="git@github.com:twisted/twisted.git";
+
+clone="$(mktemp -d -t twisted)";
+
+git clone --progress "${repo}" "${clone}";
```

Comment:
hawkowl should use the network in my office, eh?  Well, this will annoy me on my tiny Interweb straw in Canada, so OK.

`--depth 1` added in dcc8bdc5af0dfa387c1ff26a70029909f05261ca